### ISSUE TITLE
About performance of PSPNet-ResNet50

### ISSUE THREAD

**Issue by dontLoveBugs** [ycszen/TorchSeg#80]
Hi, 
I use your code to train PSPNet-ResNet50 in ade dataset and my result is:

Method |   backbone | Mean IoU | Accuracy
 :-------:| :------:  | :------: | :------:
PSPNet | resnet50v1c | 38.98% | 78.52%

The result is worst than yours. My environment is pytorch 1.0, cuda 10.

**Comment by ycszen** [comment]
Could you give me more details? 
What's the performance results with this [checkpoint](https://drive.google.com/file/d/1jDj3UJCAAffmPQ4ckTOFxQdvFWxThCoy/view)?

**Comment by dontLoveBugs** [comment]
I have tested the checkpoint and it has the same checkpoint as you shows. However, when I train the model from scratch，all my trained results are worse than yours. 
At fact, I has changed your code in order to read datasets without the train/val.txt files and added tensorboard to visulize training process. But I think these changes should not make a great difference. This is [my changed code](https://github.com/dontLoveBugs/MyTorch).

**Comment by ycszen** [comment]
Maybe you can use the `lr=2e-2`. 
I will also test the setting with the distributed-parallel api if I have the free machines.

**Comment by dontLoveBugs** [comment]
At fact, I test various parameters combinations, such as lr 0.1~0.2, weght_decay 0.0001~0.0005, with or without using apex api.
